File: QE-Selenium-Java/Activity10.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity10{
    public static void main(String[] args) {

        FirefoxDriver driver = new FirefoxDriver();
        Actions actions = new Actions(driver);

        driver.get("https://training-support.net/webelements/drag-drop");
        System.out.println(driver.getTitle());

        WebElement ball = driver.findElement(By.id("ball"));
        WebElement d1 = driver.findElement(By.id("dropzone1"));
        WebElement d2 = driver.findElement(By.id("dropzone2"));

        actions.dragAndDrop(ball, d1).build().perform();
        if (driver.findElement(By.xpath("//*[@id='dropzone1']/span")).getText().equals("Dropped!")) {
            System.out.println("Dropped in dropzone1");
        }

        actions.dragAndDrop(ball, d2).build().perform();
        if (driver.findElement(By.xpath("//*[@id='dropzone2']/span")).getText().equals("Dropped!")) {
            System.out.println("Dropped in dropzone2");
        }

        driver.quit();
    }
}

File: QE-Selenium-Python/Activity10.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains

driver = webdriver.Firefox()
actions=webdriver.ActionChains(driver)
driver.get("https://training-support.net/webelements/drag-drop")
print(driver.title)

ball = driver.find_element(By.ID, "ball")
d1 = driver.find_element(By.ID, "dropzone1")
d2 = driver.find_element(By.ID, "dropzone2")

actions = ActionChains(driver)

actions.drag_and_drop(ball, d1).perform()
if (driver.find_element(By.XPATH, "//*[@id='dropzone1']/span").text == "Dropped!"):
    print("Dropped in dropzone1")

actions.drag_and_drop(ball, d2).perform()
if (driver.find_element(By.XPATH, "//*[@id='dropzone2']/span").text == "Dropped!"):
    print("Dropped in dropzone2")

driver.quit()




